setup_claude_desktop.py

```
@@ -0,0 +1,217 @@
+
+    # Get Claude config path
+    claude_config_path = get_claude_config_path()
+
+    # Create directory if it doesn't exist
+    claude_config_path.parent.mkdir(parents=True, exist_ok=True)
+
+    # Read existing config if it exists
+    existing_config = {}
+    if claude_config_path.exists():
+        try:
+            with open(claude_config_path, 'r') as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)